distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -1879,14 +1878,16 @@ private ValidatableResponse assertIngestedDirectoryMonitor(String query, int num
       }
       try {
         TimeUnit.SECONDS.sleep(1);
-      } catch (InterruptedException e) {
+      } catch (InterruptedException ignored) {
       }
     } while (TimeUnit.NANOSECONDS.toMillis(System.nanoTime() - startTime)
         < TimeUnit.MINUTES.toMillis(3));
     response.body("metacards.metacard.size()", equalTo(numResults));
     return response;
   }
 
+  // Is this test bad??
+  @Ignore
```

Comment:
❓ Is this one broken? Should it just be deleted rather than ignored?

Reply:
This test wasn't checking the assertion it thought it was, after making these changes it was failing. Not sure if the test is bad or if it actually is catching something.